pages/test-evaluate/evaluation-tools/index.pl.md

```diff
@@ -0,0 +1,136 @@
+
+<hr>
+
+{% include excol.html type="start" id="video-intro-transcript" %}
+
+##  Omówienie narzędzi diagnostycznych {#alternatives}
+
+{% include excol.html type="middle" %}
+
+_Ten film jest również dostępny na serwerze W3C: Film: Omówienie narzędzi oceny (format pliku: MP4, rozmiar pliku: 51MB MB)](https://media.w3.org/wai/evaluation-intros/tools-for-evaluating.mp4)._
+
+###  Transkrypcja tekstowa z opisem elementów wizualnych dla filmu Wybór i stosowanie narzędzi oceny {#transcript}
+
+Filmy zawierają podstawową animację ilustrującą to, co mówi głos spoza ekranu. Ludzie są przedstawieni za pomocą symboli graficznych.
+
+<table aria-labelledby="transcript">
+  <thead>
+    <tr>
+      <th width="50%">Dźwięk</th>
+      <th>Obraz</th>
+    </tr>
+  </thead>
+<tbody>
+  <tr>
+    <td>Narzędzia oceny dostępności cyfrowej</td>
+    <td>Narzędzia oceny dostępności cyfrowej</td>
+  </tr>
+  <tr>
+    <td>Istnieją programy i usługi internetowe, które pomogą Ci rozpoznać problemy z dostępnością.</td>
+    <td>Otwiera się skrzynka narzędziowa przy komputerze. Ze skrzynki wychodzi lupa, aby sprawdzić stronę internetową na komputerze. Lupa powiększa słowo „dostępność”.</td>
+  </tr>
+  <tr>
+    <td>Mogą one zaoszczędzić wiele czasu i wysiłku podczas oceny oraz zapobiec powstawaniu nowych barier dostępności.</td>
+    <td>Oszczędzaj czas i wysiłek. Unikaj barier dostępności.</td>
+  </tr>
+  <tr>
+    <td>Jednak narzędzia nie mogą zrobić wszystkiego. Niektórych testów dostępności po prostu nie można zautomatyzować i muszą być przeprowadzone ręcznie.</td>
+    <td>Osoba obok komputera ze stroną internetową używa lupy ze słowem „dostępność”.</td>
+  </tr>
+  <tr>
+    <td>Niektóre narzędzia poprowadzą Cię przez te testy, których nie można zautomatyzować.</td>
+    <td>Lista kontrolna obok komputera.</td>
+  </tr>
+  <tr>
+    <td>Niektóre narzędzia sprawdzają tylko jedną stronę na raz, inne mogą sprawdzać całe witryny.</td>
+    <td>Strona w witrynie internetowej jest skanowana. Nad elementami strony pojawia się kolejno znak błędu, powodzenia i znak zapytania.</td>
+  </tr>
+  <tr>
+    <td>Narzędzia można integrować z różnymi środowiskami pracy. Na przykład z przeglądarką internetową, systemem zarządzania treścią (C-M-S) oraz w narzędziach programistycznych i wdrożeniowych.</td>
+    <td>Słowa „Przeglądarka internetowa”, „CMS” i „środowisko wdrożeniowe” wokół ikony narzędzi wyświetlanej na komputerze.</td>
+  </tr>
+  <tr>
+    <td>Wspierają różne role w zespole projektowym, takie jak autorów treści, twórców kodu, projektantów oraz właścicieli produktów.</td>
+    <td>Ikona narzędzi jest otoczona czterema ikonami: długopisu, kodu, pędzla i osoby z kluczem.</td>
+  </tr>
+  <tr>
+    <td>Trzeba pamiętać, że w niektórych przypadkach narzędzia mogą dawać nieprawidłowe wyniki.</td>
+    <td>Lupa powiększająca znak ostrzegawczy z wykrzyknikiem. Wyświetlanych jest wiele lup. </td>
```

Comment:
@zwiastunsw This description seems to have been reworded. Similar to https://github.com/w3c/wai-website/pull/561#discussion_r1679357937